packages/react-auth/package.json

```
@@ -28,7 +28,8 @@
   "license": "MIT",
   "packageManager": "yarn@1.22.22",
   "scripts": {
-    "build": "parcel build && cp dist/index.d.ts dist/index.d.mts",
+    "build": "yarn build-docs && parcel build && cp dist/index.d.ts dist/index.d.mts",
+    "build-docs": "yarn typedoc --plugin typedoc-plugin-rename-defaults",
```

Unnecessary `yarn` invocation.
```suggestion
    "build-docs": "typedoc --plugin typedoc-plugin-rename-defaults",
```